pkg/api/graph/test/bare-bc-can-push.yaml

```diff
@@ -0,0 +1,156 @@
+apiVersion: v1
+items:
+- apiVersion: v1
+  kind: BuildConfig
+  metadata:
+    creationTimestamp: 2015-12-16T01:01:33Z
+    labels:
+      app: repo-base
+    name: repo-base
+    namespace: repo
```

Comment:
Remove any namespaces. There are times I am running these locally and having a namespace is really annoying.